app/helpers/users_helper.rb

```diff
@@ -11,6 +11,10 @@ def full_name_and_birthdate(user)
     label
   end
 
+  def full_name(user)
```

Comment:
This method already exists in `app/models/concerns/full_name_concern.rb`

Reply:
ah cool, merci, j'avais pas fait attention ! :smile: